server/apps/immich/src/api-v1/album/album.service.spec.ts

```diff
@@ -25,6 +25,7 @@ describe('Album service', () => {
     albumEntity.createdAt = 'date';
     albumEntity.sharedUsers = [];
     albumEntity.assets = [];
+    albumEntity.albumThumbnailAssetId = null;
```

Comment:
The entity has defined `albumThumbnailAssetId` as `nullable: true`. So when TypeORM "hydrated" the entity, it would set it to `null` or `string`. This spec was not setting `albumThumbnailAssetId` to anything (`undefined`), so the type was not matching what TypeORM would "hydrate" it to, and TypeScript was complaining because I used `undefined` (instead of null) later in one of the tests for checking the expectation.